src/Controller/Component/CsrfComponent.php

```diff
@@ -120,12 +121,14 @@ public function implementedEvents()
      */
     protected function _setCookie(Request $request, Response $response)
     {
+        $expires = new Time($this->_config['expiry']);
```

Comment:
`Network\Response::_setCookies()` uses PHP's `setcookie()`. The "expire" value gets to `setcookie()` as string after putting it through `Time()->format('U')`, but casts it automatically to int. So it should be O.K.